_posts/samus_returns/2024-06-11-export-exheader.md

```diff
@@ -31,4 +33,16 @@ Next, navigate to **NCSD image options... -> Mount image to drive**, then press
 
 Press A to enter the **content0.game** directory. Highlight the **extheader.bin** file, and press A while holding R to open another menu. From here, select **Copy to 0:/gm9/out**. Copy the extracted extheader.bin file from this directory on the SD Card to some place on your computer.
 
-{% include blog/figure.html src="/assets/guides/godmode9/godmode9-exheader-cart.png" caption="Menu to extract ExHeader to output folder on SD Card" %}
\ No newline at end of file
+{% include blog/figure.html src="/assets/guides/godmode9/godmode9-exheader-cart.png" caption="Menu to extract ExHeader to output folder on SD Card" %}
+
+## CTRTool
+
+First, Download CTRTool and extract it. Open the folder with the ctrtool.exe.
+
+Hold Shift, right click into the explorer window and select Open PowerShell here.
+
+Type in `.\ctrtool.exe --exheader=exheader.bin`  with a space at the end and then Drag & Drop your Metroid: Samus Returns 3DS file into the PowerShell window. It should say something like `.\ctrtool.exe --exheader=exheader.bin C:\ctrtool\MSR.3ds`.
```

Comment:
Can you rewrite these paragraphs to be a bit less windows focused? There are releases for linux and macos too.

Reply:
I have no idea how those os' work.

Reply:
Basically the same as windows in this regard. You open a terminal and execute the ctrtool binary.